README.md

```diff
@@ -2,8 +2,7 @@
 This is a helm plugin that installs raw helm charts from github.
 
 # Installation
-  * `cd $HELM_HOME/plugins/`
-  * `git clone -b master https://github.com/sagansystems/helm-github.git`
+  * `helm plugin install https://github.com/sagansystems/helm-github.git`
```

Comment:
Could you provide an example of installing a branch reference instead of implicit master so that its clear how to install the plugin for development.

Reply:
![image](https://user-images.githubusercontent.com/52489/33590100-fa79e052-d931-11e7-9879-b0fd7db7d09a.png)